instrumentation/opentelemetry-instrumentation-confluent-kafka/src/opentelemetry/instrumentation/confluent_kafka/utils.py

```diff
@@ -114,32 +121,29 @@ def _enrich_span(
     topic,
     partition: Optional[int] = None,
     offset: Optional[int] = None,
-    operation: Optional[MessagingOperationValues] = None,
+    operation: Optional[MessagingOperationTypeValues] = None,
 ):
     if not span.is_recording():
         return
 
-    span.set_attribute(SpanAttributes.MESSAGING_SYSTEM, "kafka")
+    span.set_attribute(MESSAGING_SYSTEM, "kafka")
     span.set_attribute(SpanAttributes.MESSAGING_DESTINATION, topic)
-
     if partition is not None:
-        span.set_attribute(SpanAttributes.MESSAGING_KAFKA_PARTITION, partition)
```

Comment:
You changed the attribute here, it's not the only one and I won't add a comment for every one you changed.